### PR TITLE
chore: figures

### PR DESCRIPTION
**Pull request by georgosgeorgos**
* Add figures README.
     * Case study.  
     * Graphical abstract for discovery workflow.

* Add figures to main docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,9 @@
 [![DOI](https://zenodo.org/badge/458309249.svg)](https://zenodo.org/badge/latestdoi/458309249)
 [![2022 IEEE Open Software Services Award](https://img.shields.io/badge/Award-2022%20IEEE%20Open%20Software%20Services%20Award-yellow)](https://conferences.computer.org/services/2022/awards/oss_award.html)
 
-<img src="./docs/_static/gt4sd_logo.png" alt="logo" width="500"/>
+<img src="./docs/_static/gt4sd_logo.png" alt="logo" width="300"/>
```
Comment (jannisborn) [inline README.md:15]:
```suggestion
<img align="right" width="200" height="200" src="./docs/_static/gt4sd_logo.png" alt="logo">
```

File: README.md [hunk 2]
```diff
@@ -12,7 +12,9 @@
 [![DOI](https://zenodo.org/badge/458309249.svg)](https://zenodo.org/badge/latestdoi/458309249)
 [![2022 IEEE Open Software Services Award](https://img.shields.io/badge/Award-2022%20IEEE%20Open%20Software%20Services%20Award-yellow)](https://conferences.computer.org/services/2022/awards/oss_award.html)
 
-<img src="./docs/_static/gt4sd_logo.png" alt="logo" width="500"/>
+<img src="./docs/_static/gt4sd_logo.png" alt="logo" width="300"/> 
+<img src="./docs/_static/gt4sd_case_study.png" alt="logo" width="500"/>
```
Comment (jannisborn) [inline README.md:16]:
```suggestion
<p align="center">
	<img src="./docs/_static/gt4sd_case_study.png" alt="photo not available" width="100%" height="100%">
</p>```

**Review by jannisborn — APPROVED** [review]
Very nice, great proactivity, I just suggested some comestics for the readme formatting

**Review by drugilsberg — COMMENTED** [review]
I would consider removing the logo and swapping the order of the case study and the main figure. Also the resolution is not amazing, so we could reduce a bit the size of the images.

**Comment by georgosgeorgos** [comment]
> I would consider removing the logo and swapping the order of the case study and the main figure. Also the resolution is not amazing, so we could reduce a bit the size of the images.

For the resolution: I converted pdf to png. Not sure if we can use the pdf directly in markdown.

**Review by drugilsberg — CHANGES_REQUESTED** [review]
Structure looks good, but image quality is terrible. How many DPIs are we using here?

**Review by drugilsberg — COMMENTED** [review]
To me it looks really good, but I see a problem with mypy now, maybe some library has been updated in the meanwhile.

**Review by drugilsberg — APPROVED** [review]
🚀

**Pull request merged**